app/routes/sliderForm.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import SliderQuestions from '~/components/SliderQuestions';
+
+const Slider: React.FC = () => {
+    const [currentSymptom, setCurrentSymptom] = useState('Depression'); // change symptom state to get different questions outlined below
+
+    const handleSliderChange = (value: number) => {
+        console.log("Slider Value:", value);
+    };
+
+    const symptomQuestions = {
```

Comment:
I think symptom questions might be better stored outside of the parent form component (e.g. as a seperate json)